### PR TITLE
Brought in changes from InfiniteRecharge repo

### PR DESCRIPTION
**Pull request by qiandrewj**
### Purpose
The purpose of this pull request was to clean up the code in the Octabot repo so that it was up to date with InfiniteRecharge
### Description

- Edited bracketing
- Replaced Targeting subsystem in Octabot repo with the updated Targeting file from InfiniteRecharge
### Testing
Code builds but is not tested.

### CODE REVIEW

**Review by bjmcternan — COMMENTED** [review]
Looks good. Let's test this before merging it

**Review by ghost — CHANGES_REQUESTED** [review]
This looks good to me. Built but not tested. Will approve when tested.

Edit: Has been tested. Approved.

**Comment by bjmcternan** [comment]
@NotVinnny now that the robot is working, please change you review to approved. @qiandrewj once Vincent approves, please merge it in and delete the branch

**Review by ghost — APPROVED** [review]

**Pull request merged**
